src/main/webapp/js/chart-loader.js

```diff
@@ -0,0 +1,31 @@
+
+google.charts.load("current", {packages: ["corechart"],});
+google.charts.setOnLoadCallback(
+  function() {
+    drawChartForDiffTypesOfTea();
+  });
+
+(function drawChartForDiffTypesOfTeaForDiffTypesOfTea() {
+  var diffTypeOfTea = new google.visualization.DataTable();
+  //define columns for the DataTable instance
+  diffTypeOfTea.addColumn("string", "Types of Tea");
+  diffTypeOfTea.addColumn("number", "Cupes Of Tea");
+  //add temp data to diffTypeOfTea chart
+  diffTypeOfTea.addRows([
+    ["Green Tea", 6],
+    ["Black Tea", 10],
+    ["Oolong Tea", 7],
+    ["Matcha Tea", 4],
+    ["White Tea", 8]
+  ]);
+  //cutomization to the chart
+  var chartOptions = {
+    'title': 'Different Types of Tea We Drink in a Day',
+    width: 800,
+    height: 400
+  };
+  var chart = new google.visualization.BarChart(
+    document.getElementById("bookChart")
+  );
+  chart.draw(diffTypeOfTea, chartOptions);
+})();
```

Comment:
By calling this here, you are causing the function to be invoked immediately, even before `google.charts.load` finishes loading the JavaScript dependencies. That may result in the symbols you use (e.g. `google.visualization.DataTable`, `google.visualization.BarChart`) not yet being valid / existing when this runs and causing an error.